### PR TITLE
 Fix query for multiple delta fields in case of simple watermark (cont. of #58) 

### PR DESCRIPTION
**Pull request by lbendig**
When ```source.querybased.watermark.type=simple``` and ```extract.delta.fields``` contains a comma separated list of fields (e.g: id, type), one of the initial MySQL query looks like this during the job initialization:

```SELECT max(COALESCE(id,type)) FROM test  where  (id,type >= 477 and id,type <= 544);```

The query syntax seems to be invalid which leads to the following error:
```java.sql.SQLException: Operand should contain 1 column(s)```

After checking the generated queries of other watermark types, I suspect that here the fields in the where clause should be coalesced too (e.g: COALESCE(id,type) >=477...etc)
If this is the case (and the where clause is not intended to be a row subquery for instance) then the patch would address this issue.

### CODE REVIEW

File: gobblin-core/src/main/java/gobblin/source/extractor/utils/Utils.java [hunk 1]
```diff
@@ -53,7 +53,20 @@ public static String getClause(String clause, String datePredicate) {
     }
     return retStr;
   }
-
+  
+  /**
+   * Get coalesce of columns if there are multiple comma-separated columns
+   */
+  public static String getCoalesceColumnNames(String columnOrColumnList) {
+    if (columnOrColumnList == null) {
```
Comment (liyinan926) [inline gobblin-core/src/main/java/gobblin/source/extractor/utils/Utils.java]: Better to use `Strings.isNullOrEmpty(columnOrColumnList)` instead.

**Comment by liyinan926** [comment]
LGTM. @narasimhareddyv, can you take a look? Thanks!

**Comment by narasimhareddyv** [comment]
Looks good.

**Comment by lbendig** [comment]
@liyinan926 @narasimhareddyv Thank you!

**Pull request merged**